datafusion/core/tests/parquet/filter_pushdown.rs

```diff
@@ -600,3 +617,149 @@ fn get_value(metrics: &MetricsSet, metric_name: &str) -> usize {
         }
     }
 }
+
+///  a ParquetFile that has been created for testing.
+pub struct TestParquetFile {
```

Comment:
this is a copied subset of https://github.com/apache/arrow-datafusion/blob/main/parquet-test-utils/src/lib.rs